src/views/core/bodies/ViewBoards.vue

```diff
@@ -109,7 +128,7 @@ export default {
     },
     fetchData () {
       // Get boards of the body
-      this.axios.get(this.services['network'] + '/bodies/' + this.$route.params.id + '?sort=start_date&direction=desc').then((response) => {
+      this.axios.get(this.services['network'] + '/bodies/' + this.$route.params.id + '/boards?sort=start_date&direction=desc').then((response) => {
```

Comment:
I think I need to merge https://github.com/AEGEE/network/pull/599 before this works. Maybe move the changes to this file to another PR so we can merge the actual AC checks first

Reply:
Oh wait, it needs that PR either way to properly display the AC page. We can keep this

Reply:
Ah I do agree that it would have been cleaner to make this a separate PR